doc/reduce-memory.md

```diff
@@ -18,7 +18,7 @@ The size of some in-memory caches can be reduced. As caches trade off memory usa
 
 - Since `0.14.0`, unused memory allocated to the mempool (default: 300MB) is shared with the UTXO cache, so when trying to reduce memory usage you should limit the mempool, with the `-maxmempool` command line argument.
 
-- To disable most of the mempool functionality there is the `-blocksonly` option. This will reduce the default memory usage to 5MB and make the client opt out of receiving (and thus relaying) transactions, except from whitelisted peers and as part of blocks.
+- To disable most of the mempool functionality there is the `-blocksonly` option. This will reduce the default memory usage to 5MB and make the client opt out of receiving (and thus relaying) transactions, except from peers who have `NetpermissionFlags::Relay` set (e.g. whitelisted peers), and as part of blocks.
```

Comment:
nit: Might be good to use the value referred to by the RPC or manpage, not the internal fragile symbol name

Reply:
OK I switched to the RPC permission.